process/sync/shardblock.go

```diff
@@ -144,7 +144,12 @@ func (boot *ShardBootstrap) StartSyncingBlocks() {
 func (boot *ShardBootstrap) SyncBlock(ctx context.Context) error {
 	err := boot.syncBlock()
 	if errors.IsGetNodeFromDBError(err) {
```

Comment:
You should check if err is of type `GetNodeFromDBErr` inside the `errors.IsGetNodeFromDBError()` func.